# Genetic Capitalism and Stabilizing Selection of Antimicrobial Resistance Genotypes in _Escherichia coli_

|                                              Authors                                              |    Paper    |   Citation  | Figures |
|-------------------------------------------------------------------------------------------------|:-----------:|:-----------:|:--------:|
| Colby T. Ford, Ph.D.<br>Kevin Smith<br>Gabriel Lopez Zenarosa, Ph.D.<br>David Brown<br>John Williams<br>Daniel Janies, Ph.D. | [Coming Soon - Cladistics](#) | ```Coming Soon``` | [Tableau Viz](https://public.tableau.com/profile/cford38#!/vizhome/E_coliGenotypeSetsViz/GeneticCapitalism) |

## Abstract
Antimicrobial resistance (AMR) in pathogenic strains of bacteria, such as _Escherichia coli_ (_E. coli_), adversely impact personal and public health. In this study, we examine competing hypotheses for the evolution of AMR including: 1) “genetic capitalism” in which genotypes that confer antibiotic resistance are gained and not often lost in lineages, and 2) “stabilizing selection” in which genotypes that confer antibiotic resistance are gained and lost often. To test these hypotheses, we assembled a dataset that includes annotations for 409 AMR genotypes and a phylogenetic tree based on genome-wide single nucleotide polymorphisms from 29,255 isolates of _E. coli_. We used phylogenetic methods to count the times each AMR genotype was gained and lost across the tree and used model-based clustering of the genotypes with respect to their gain and loss rates. We demonstrate that many genotypes cluster to support the hypothesis for genetic capitalism while a few cluster to support the hypothesis for stabilizing selection. Comparing the sets of genotypes that fall under each of the hypotheses, we found a statistically significant difference in the breakdown of resistance mechanisms through which the AMR genotypes function. The result that many AMR genotypes cluster under genetic capitalism reflects that strong positive selective forces, primarily induced by human industrialization of antibiotics, outweigh the potential fitness costs to the bacterial lineages for carrying the AMR genotypes. We expect genetic capitalism to further drive bacterial lineages to resist antibiotics. We find that antibiotics that function via replacement and efflux tend to behave under stabilizing selection and thus may be valuable in an antibiotic cycling strategy.

-------------------------------------
## Supplemental Materials

|     | File Type        | File Name                                                                                                | Description                                                                                                                                                          |
|-----|------------------|----------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| 1.  | Script           | [clustering.R](clustering.R)                                                                             | Script for performing cluster analysis of AMR genotype gains and losses.                                                                                             |
| 2.  | Script           | [distribution_tests.R](distribution_tests.R)                                                             | Script for performing distribution tests by AMR group & resistance mechanism.                                                                                        |
| 3.  | Script           | [TNT/TNTFileGenerator/TNT_file_generator.R](TNT/TNTFileGenerator/TNT_file_generator.R)                   | Script for generating the data file for the TNT-based phylogenetic analysis.                                                                                         |
| 4.  | Reference Data   | [TNT/PDG000000004.1024.reference_target.tree.newick](TNT/PDG000000004.1024.reference_target.tree.newick) | Reference Tree from NCBI. (Current PDG trees for _E. coli_ can be found [here](https://ftp.ncbi.nlm.nih.gov/pathogen/Results/Escherichia_coli_Shigella/).)                |
| 5.  | Reference Method | [NCBI_TreeMethods.txt](NCBI_TreeMethods.txt)                                                             | Information of the NCBI processing pipeline for generating the SNP trees. (Can also be found on NCBI's site [here](https://ftp.ncbi.nlm.nih.gov/pathogen/Methods.txt).) |
| 6.  | Reference Data   | [IsolationData.xlsx](IsolationData.xlsx)                                                                 | Roll-Up of Isolation Metadata (Source country, organism, etc.)                                                                                                       |
| 7.  | Reference Data   | [AMR_FunctionalMechanisms.csv](AMR_FunctionalMechanisms.csv)                                             | Functional mechanisms of the 409 AMR genotypes in this study. (From [CARD](https://card.mcmaster.ca/))                                                               |
| 8.  | Results          | [clusters.csv](clusters.csv)                                                                             | Resulting AMR genotype clusters from the cluster analysis of gains and losses.                                                                                       |
